github-page/_guides/advanced/objectId.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Using ObjectId"
+redirect_from:
+  - /guides/advanced/objectid
+---
+
+Using the type `ObjectId` can be done by defining it as such:
+
+```ts
+import mongoose from 'mongoose';
+import { prop } from '@typegoose/typegoose';
+
+class FooModel {
+  @prop()
+  userId: mongoose.Schema.Types.ObjectId;
+}
+```
+
+You need to refer to the full length type since defining it as `type ObjectId = mongoose.Schema.Types.ObjectId` and referencing that will lead to it being an `Object` at compile time meaning Typegoose will translate the property type to `Mixed`.
```

Comment:
```suggestion
You need to refer to the full length type since defining it as `type ObjectId = mongoose.Schema.Types.ObjectId` and referencing that will lead to it being an `Object` at compile time, meaning Typegoose will translate the property type to `Mixed`.
```

Reply:
Fixed!